src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/AddStatisticsCardView/AddStatisticsCardsViewModel.swift

```diff
@@ -0,0 +1,101 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import OpenCombine
+
+class AddStatisticsCardsViewModel {
+	
+	// MARK: - Init
+	
+	init(
+		localStatisticsModel: AddLocalStatisticsModel,
+		presentSelectStateList: @escaping (SelectValueViewModel) -> Void,
```

Comment:
shouldn't it be presentFederalStatesList?

Reply:
What is the difference between a State and federalState ?

Reply:
A state can be a state in the app, so not to confuse the new developer, also it's "plural" States as is a list of FederalStates